CONFIGDOC.md

```diff
@@ -239,7 +245,8 @@ pipeline {
             steps
             {
                 runMATLABTests(testResultsJUnit: 'test-results/results.xml',
-                               codeCoverageCobertura: 'code-coverage/coverage.xml')
+                               codeCoverageCobertura: 'code-coverage/coverage.xml',
+                               sourceFolder: 'source')

```

Comment:
Even for a single value being passed to sourceFolder, user needs to include square brackets. Something to the effect of,

sourceFolder: ['source']

else there's a type mismatch error.

Reply:
Unfortunate :(. Especially for MATLAB users. Is there a way we can support both scalar and array values?

Reply:
Fixed the examples.